models/response_entities.go

```diff
@@ -19,17 +19,18 @@ type CreatedInsolvencyResourceLinks struct {
 
 // CreatedPractitionerResource is the entity returned in a successful creation of an practitioner resource
 type CreatedPractitionerResource struct {
-	PractitionerId  string                       `json:"practitioner_id"`
-	IPCode          string                       `json:"ip_code"`
-	FirstName       string                       `json:"first_name"`
-	LastName        string                       `json:"last_name"`
-	Email           string                       `json:"email"`
-	TelephoneNumber string                       `json:"telephone_number"`
-	Address         CreatedAddressResource       `json:"address"`
-	Role            string                       `json:"role"`
-	Etag            string                       `json:"etag"`
-	Kind            string                       `json:"kind"`
-	Links           PractitionerResourceLinksDao `json:"links"`
+	PractitionerId  string                        `json:"practitioner_id"`
+	IPCode          string                        `json:"ip_code"`
+	FirstName       string                        `json:"first_name"`
+	LastName        string                        `json:"last_name"`
+	Email           string                        `json:"email"`
+	TelephoneNumber string                        `json:"telephone_number"`
+	Address         CreatedAddressResource        `json:"address"`
+	Role            string                        `json:"role"`
+	Etag            string                        `json:"etag"`
+	Kind            string                        `json:"kind"`
+	Links           PractitionerResourceLinksDao  `json:"links"`
+	Appointment     AppointedPractitionerResource `json:"appointment"`
```

Comment:
Should Appointment be 'omitempty' here?

Reply:
Discussed.Thanks

Reply:
Using 'omitempty' is now necessary as you had pointed previously in order to support possible nil appointment, having changed the logics to be returning practitioners with no appointment, so long there is one with appointment.